fbbt/src/ontologies/fbbt-edit.obo

```diff
@@ -63575,6 +63574,260 @@ intersection_of: FBbt:00005106 ! neuron
+is_a: FBbt:00040006 ! synaptic neuropil subdomain
+relationship: part_of FBbt:00001962 ! embryonic/larval tritocerebrum
+property_value: created_by "dos" xsd:string
+creation_date: 2018-02-15T10:53:13Z
+
+[Term]
+id: FBbt:00007721
+name: embryonic/larval superior inferior protocerebrum
+def: "A small synaptic neuropil of the larval brain that has the shape of a hemi-cylinder.  It flanks the mushroom body vertical lobe laterally and posteriorly,  where it wedges in between SLP (superior lateral protocerebrum) and SMPp. It borders the IPl and (briefly) IPa compartments and (very anteriorly and ventrally) the Spur." [FBC:dw]
+is_a: FBbt:00040007 ! synaptic neuropil domain
+relationship: part_of FBbt:00001925 ! embryonic/larval protocerebrum
+property_value: created_by "dos" xsd:string
+creation_date: 2018-02-15T11:10:38Z
+
+[Term]
+id: FBbt:00007722
+name: embryonic/larval nodulus - primordial
+def: "A small primordium that appears as a ventrally directed process of the  lateral  primordial fan-shaped body.  It is demarcated from the surrounding Inferior medial and anterior protocerebrum by different the different concentration of synapses (e.g. as assayed by Bruchpilot expression).  It develops from elements of the DPMm1, DPMpm1/2 and CM4 lineages." [FBC:dw, FlyBase:FBrf0211016]
+is_a: FBbt:00040007 ! synaptic neuropil domain
+property_value: created_by "dos" xsd:string
+creation_date: 2018-02-15T14:21:50Z
+
+[Term]
+id: FBbt:00007723
+name: embryonic/larval posterior lateral protocerebrum
+def: "A synpatic neuropil domain located ventro-postero-laterally in the larval central brain. Anteriorly, the great commissure (BLAv1,BLD5) defines the boundary, in the coronal plane, with the Ventolateral protocerebrum (VLP). Dorsally, there is a slim border with the superior lateral protocerebrum (SLP).   Medially, a virtual sagittal plane intersecting the CP1v tract and the Balp2/3 tract separates the this domain from the ventromedial cerebrum (VMCp).   Laterally, a glial septum and BLP1/2 tract separates it from the lobula." []
+comment: Developmental relation to adult synaptic neuropil domain is a PC from Volker Hartenstein. Note - description of relative location of brain structures in this definition is based on Younossi-Hartenstein et al., 2003 description of the first instar larval brain and Pereanu et al., 2010 third instar larval brain.
+synonym: "centro-posterior medial synaptic neuropil domain" EXACT []
+synonym: "centroposterior medial compartment" EXACT [FlyBase:FBrf0155900]
+synonym: "CPM" EXACT [FlyBase:FBrf0155900]
+synonym: "larval IPp" NARROW [FlyBase:FBrf0211016]
+synonym: "larval posterior inferior protocerebrum" NARROW [FlyBase:FBrf0211016]
+xref: VFB:FBbt_00007070
+is_a: FBbt:00040007 ! synaptic neuropil domain
+relationship: part_of FBbt:00001925 ! embryonic/larval protocerebrum
+property_value: created_by "dos" xsd:string
+creation_date: 2018-02-15T14:33:12Z
+
```

Comment:
**embryonic/larval posterior ventromedial cerebrum**
The posterior part of the ventromedial cerebrum (BPM). It borders the posterior Inferior Protocerebrum (IPp) dorso-medially, the boundary defined by the DPPT(DPMl1) fascicle and a virtual axial plane drawn from the MEF medially to the medial neuropil edge.  The medial Clamp is bordered dorso-laterally of the VMCp, the boundary given as a virtual axial plane intersecting the MEF(CM1,3,4, medially) and LEFp(CP1v, laterally) fascicles (Hartenstein et al., 2015).  The Posterior Lateral Protocerebrum (PLP) is lateral, the boundary a virtual sagittal plane intersecting the LEFp (CP1v; posteriorly) and the loVL (Balp2/3; anteriorly) fascicles.  The VMCp is posterior of the anterior ventromedial cerebrum (VMCa), the split define by the coronal slice level of the Great Commissure, GC (BLAv1,BLD5).